test/PizzeriaTest.cpp

```diff
@@ -65,3 +64,28 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTest, completeOrderWithStubPizzaAndTwoMockPizzas) {
+    StrictMock<PizzaMock> strict;
+    NiceMock<PizzaMock> nice;
+    PizzaStub stub{"hawaii"};
+    EXPECT_CALL(strict, getName())
+        .WillOnce(Return("americana"));
+    EXPECT_CALL(strict, getPrice())
+        .WillOnce(Return(30));
+    EXPECT_CALL(strict, getBakingTime())
+        .WillOnce(Return(minutes(10)));
+    EXPECT_CALL(nice, getName())
+        .WillOnce(Return("inferno"));
+    EXPECT_CALL(nice, getPrice())
+        .WillOnce(Return(50));
+    EXPECT_CALL(nice, getBakingTime())
+        .WillOnce(Return(minutes(20)));
```

Comment:
You have magic values - 30, 50, minutes(20). Use constexpr for them. It shows the relation between objects because you will use them in proper EXPECT_CALLs or ASSERT_EQ.